### PR TITLE
code style

### PR DESCRIPTION
**Pull request by Hojun-Cho**
I found other things that needed fixing, so I recreated the PR.
modified: 'if('  =>  'if ('. Is okay?

### CODE REVIEW

File: src/internal/qinternal.h [hunk 1]
```diff
@@ -41,15 +41,15 @@
         size_t _strsize;                                                \
         for(_strsize = 1024; ; _strsize *= 2) {                         \
```
Comment (wolkykim) [inline src/internal/qinternal.h:42]: do we want to style this the same way?

File: src/internal/qinternal.h [hunk 2]
```diff
@@ -41,15 +41,15 @@
...
             s = (char*)malloc(_strsize);                                \
-            if(s == NULL) {                                             \
+            if (s == NULL) {                                             \
```
Comment (wolkykim) [inline src/internal/qinternal.h:44]: we want the tailing `\` lineup with other lines

**Comment by wolkykim** [comment]
thanks for the PR. I left a few comments. Have you checked other files? Are those the only files needing style adjustment?

**Comment by wolkykim** [comment]
sorry clicked the wrong button. reopened.

**Comment by Hojun-Cho** [comment]
> thanks for the PR. I left a few comments. Have you checked other files? Are those the only files needing style adjustment?

Maybe we should style the comments too? If not, I think we're done.

File: src/internal/qinternal.h [hunk 3]
```diff
@@ -39,17 +39,17 @@
 
 #define DYNAMIC_VSPRINTF(s, f) do {                                     \
         size_t _strsize;                                                \
-        for(_strsize = 1024; ; _strsize *= 2) {                         \
+        for (_strsize = 1024; ; _strsize *= 2) {                         \
```
Comment (wolkykim) [inline src/internal/qinternal.h:42]: the tailing `\` are off lining up with others.

File: src/internal/qinternal.h [hunk 4]
```diff
@@ -39,17 +39,17 @@
...
+            if (s == NULL) {                                             \
                 DEBUG("DYNAMIC_VSPRINTF(): can't allocate memory.");    \
                 break;                                                  \
             }                                                           \
             va_list _arglist;                                           \
             va_start(_arglist, f);                                      \
             int _n = vsnprintf(s, _strsize, f, _arglist);               \
             va_end(_arglist);                                           \
-            if(_n >= 0 && _n < _strsize) break;                         \
+            if (_n >= 0 && _n < _strsize) break;                         \
```
Comment (wolkykim) [inline src/internal/qinternal.h:52]: the tailing `\` are off lining up with others here too. This is happening for many of the updates. Please check them again. Maybe easier to see it on the diff UI https://github.com/wolkykim/qlibc/pull/103/files

**Review by wolkykim — CHANGES_REQUESTED** [review]

**Comment by Hojun-Cho** [comment]
Fixed. Thanks for letting me know. Are the changes correct?

**Comment by Hojun-Cho** [comment]
Fixed. Thanks for letting me know. Are the changes correct?

File: src/internal/qinternal.h [hunk 5]
```diff
@@ -72,63 +72,63 @@ struct qmutex_s {
...
+        }                                                                    \
+        int _ret = pthread_mutex_init(&(x->mutex), &_mutexattr);             \
+        pthread_mutexattr_destroy(&_mutexattr);                              \
+        if (_ret == 0) {                                                     \
+            m = x;                                                           \
+        } else {                                                             \
+            DEBUG("Q_MUTEX: can't initialize mutex. [%d]", _ret);            \
+            free(x);                                                         \
+            m = NULL;                                                        \
+        }                                                                    \
```
Comment (wolkykim) [inline src/internal/qinternal.h:90]: You only need to update L79 and L84.

**Review by wolkykim — COMMENTED** [review]
What's your plan for other files? Are you going to work on applying this convention to all the other code and header files? Or partially only to those files included in this PR?

**Comment by Hojun-Cho** [comment]
I created the PR because I enjoy looking at qlibc code. I don't plan to apply this rule to any other files (though I will if you need to).. am I messing with qlibc code?

**Comment by wolkykim** [comment]
That's nice. And no, you're not messing with the code but making it better. I appreciate your work. Yeah, making everything consistent would be nice but it's a large work.

**Review by wolkykim — APPROVED** [review]

**Comment by Hojun-Cho** [comment]
Thank you for creating such a great library ^%^.

**Pull request merged**